examples/using-remark/src/pages/2016-04-15---hello-world-kitchen-sink/index.md

```diff
@@ -133,8 +133,8 @@ In this example, leading and trailing spaces are shown with with dots: ⋅
 
   You can have properly indented paragraphs within list items. Notice the blank line above, and the leading spaces (at least one, but we'll use three here to also align the raw Markdown).
 
-  To have a line break without a paragraph, you will need to use two trailing spaces.  
```

Comment:
Ah, there even is an example for the two space stuff – totally forgot abt that already :)